Roommate-Budget-Helper/budget-helper-webui
# Created landing page, added higher quality images

## Jira Ticket:
RBH-63-landing-page

## Description of Work
Added a landing page with some welcome text and a button that routes users to the login page.

## Acceptance Criteria
Routes users to login page on button click

-

## How to Test
go to [localhost:3000](url) , then observe the page. Click on the login/register button, and verify that you are on the login page

-

### List of Changes:
Made landing content for the index page. Added an image for the landing page and added a higher quality image for the logo.
-

### Screenshots

| Image | Description |
| ----- | ----------- |
|![image](https://user-images.githubusercontent.com/50157964/230028020-44f09252-c4b4-4e59-afe5-3494393519d8.png)|The landing page|
